packages/property-grid/src/components/ElementList.tsx

```diff
@@ -192,6 +194,7 @@ export const ElementList = ({
       </div>
       <div className="property-grid-react-element-list-container">
         {dataProvider && (
+          /* eslint-disable-next-line deprecation/deprecation */
```

Comment:
future upgrade to iTwin UI I assume?

Reply:
yea i was going to tackle that next week if i get time